plugins/org.eclipse.fordiac.ide.fbtypeeditor.ecc/src/org/eclipse/fordiac/ide/fbtypeeditor/ecc/properties/ActionSection.java

```diff
@@ -160,24 +159,19 @@ public void setInput(final IWorkbenchPart part, final ISelection selection) {
 	}
 
 	@Override
-	public void refresh() {
-		final CommandStack commandStackBuffer = commandStack;
-		commandStack = null;
-		if ((null != type) && (null != getFBType())) {
-			// during delete phases it can be that the input (i.e., Action) is not attached
-			// to its type anymore. Therefore also the check if getFBType() is not null
-			updateDropdown(outputEventCombo, ECCContentAndLabelProvider.getOutputEventNames(getFBType()));
-			selectOutputEvent(getType().getOutput());
+	protected void performRefresh() {
+		// during delete phases it can be that the input (i.e., Action) is not attached
```

Comment:
You kept the comment but not the if(null!=getFBType)

Reply:
thx for noting. brought it back.